app/serializers.py

```diff
@@ -131,3 +131,20 @@ class Meta:
 
     def get_avatar(self, data):
         return f"{settings.DOMAIN_NAME}{data.image.url}"
+
+
+class UserPasswordSerializer(serializers.Serializer):
+    password = serializers.CharField(min_length=8, required=True)
+    old_password = serializers.CharField()
+
+    def validate(self, data):
+        old_password = data["old_password"]
+        password = data["password"]
+        if not (
```

Comment:
raise two different messages depending on the check

Reply:
Password does not match
New password cannot be same as the old password.

Reply:
done